src/main/resources/spring/migration-bean.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+       http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd">
+
+    <context:annotation-config/>
+
+    <context:component-scan base-package="org.fcrepo"/>
+<!--  
+    <bean class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
+        <property name="location">
+            <value>Migrator.properties</value>
+        </property>
+    </bean> 
+ -->
+
+    <bean id="directoryScanningIDResolver" class="org.fcrepo.migration.foxml11.DirectoryScanningIDResolver">
+        <constructor-arg value="target/index"/>
+        <constructor-arg value ="src/test/resources/datastreamStore" />
+    </bean>
+
+    <bean id="httpClientURLFetcher" class="org.fcrepo.migration.foxml11.HttpClientURLFetcher">
+    </bean>
+ 
+    <bean id="foxmlDirectoryObjectSource" class="org.fcrepo.migration.foxml11.FoxmlDirectoryObjectSource" init-method="init">    
+        <property name="resolver" ref="directoryScanningIDResolver"/>
+        <property name="fetcher" ref="httpClientURLFetcher"/>
+        <property name="root" value="src/test/resources/objectStore"/>
+        <property name="objectStore" value="src/test/resources/objectStore"/>
+        <property name="datastreamStore" value="src/test/resources/datastreamStore"/>
+        <property name="workingDir" value="target/index"/>
+     </bean>
+ 
+    <bean id="consoleLoggingStreamingFedoraObjectHandler" class="org.fcrepo.migration.handlers.ConsoleLoggingStreamingFedoraObjectHandler">
```

Comment:
Instead of defining a bean for this ObjectHandler, you should be able to take advantage of the "context:component-scan" above and annotate the "ConsoleLoggingStreamingFedoraObjectHandler" with "@Component"